src/test/resources/api/products/_get/response.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "skip",
+  "title": "type:String",
+  "description": "type:String",
+  "price": "regex:[\\d]*",
+  "discountPercentage": "regex:[\\d]+.[\\d]*",
+  "rating": "regex:[\\d]+.[\\d]*",
+  "stock": "regex:[\\d]",
+  "brand": "type:String",
+  "category": "type:String",
+  "thumbnail": "type:String",
+  "images": [
+    "type:String"
+  ]
```

Comment:
Please let me know if my understanding is correct. We use schema when we only want to validate structure and use response.json when we want to validate some values for the fields. So, in such case we could skip fields.